### ISSUE TITLE
Upgrade to v1.21.2+k0s.0 fails when using the default config 

### ISSUE THREAD

**Issue by mviitane** [k0sproject/k0s#1000]
**Version**
```
$ k0s version
v1.21.2+k0s.0
```

**Platform**
```
$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.2 LTS
Release:	20.04
Codename:	focal
```

**What happened?**
Upgrade to v1.21.2+k0s.0 fails when using the default config (k0sctl or local k0s upgrade)

**How To Reproduce**
v1.21.2+k0s.0 fails to start when having the following parameter in the configuration
`spec.telemetry.interval`

This parameter was part of the default config in the previous releases.

**Workaround**
Remove this parameter from the k0s configuration before upgrading to v1.21.2+k0s.0.
  `spec.telemetry.interval`

When using k0sctl, k0s configuration file can be found from /etc/k0s/k0s.yaml from the controller nodes.

**Comment by mviitane** [comment]
Patch release is being prepared to fix this.

**Comment by mviitane** [comment]
Fixed with #995, included into v1.21.2+k0s.1
